src/mixins/api-error-mixin.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { AxiosError } from 'axios'
+
+@Component
+export class ApiErrorMixin extends Vue {
```

Comment:
Another new mixin with `handleApiError()` for everyone to use.

Reply:
I think I got rid of the other handleApiError but please verify.

Reply:
This didn't work ("cannot get components of undefined").
This file is currently named "~api-error-mixin.vue" to denote that it's not being used (and it's commented out in the index.ts).